src/main/java/seedu/address/logic/parser/ParserUtil.java

```diff
@@ -95,6 +103,66 @@ public static Year parseYear(String year) throws ParseException {
         return new Year(year);
     }
 
+    /**
+     * Parses a {@code String venue} into a {@code ClassVenue}.
+     * Leading and trailing whitespaces will be trimmed.
+     *
+     * @throws ParseException if the given {@code venue} is invalid.
+     */
+    public static ClassVenue parseVenue(String venue) throws ParseException {
+        requireNonNull(venue);
+        String trimmedVenue = venue.trim();
+        if (!ClassVenue.isValidClassVenue(trimmedVenue)) {
+            throw new ParseException(Name.MESSAGE_CONSTRAINTS);
```

Comment:
Should this be `ClassVenue.MESSAGE_CONSTRAINTS`?

Reply:
Oops. My bad. Thanks for pointing that out!